wifiphisher/extensions/deauth.py

```diff
@@ -142,7 +142,7 @@ def get_packet(self, packet):
             receiver = packet.addr1
             sender = packet.addr2
         except AttributeError:
-            logger.warning("A malformed packet was discarded")
+            logger.debug("malformed frame doesn't contain address fields")
```

Comment:
I changed it to the `debug level` since it's a normal behavior (i.e. monitor mode will receive the frames which have the wrong crc value)